### ISSUE TITLE
Release iOS version

### ISSUE THREAD

**Issue by arnabk** [binary-com/tradingview#6]
<a href="https://github.com/arnabk"><img src="https://avatars.githubusercontent.com/u/446864?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [arnabk](https://github.com/arnabk)**
_Wednesday Aug 12, 2015 at 06:58 GMT_
_Originally opened as https://github.com/regentmarkets/tradingview/issues/9_

----

**Comment by arnabk** [comment]
Under discussion with JY. With Mozilla planning to release their iOS version of Firefox, https://www.mozilla.org/en-US/firefox/ios/ , I do not think we need this app. Firefox app should work through Firefox browser in iOS devices.

**Comment by borisyankov** [comment]
May be not. Just as with Chrome on iOS, Firefox will be running webkit internally. Apple does not allow another browser engine on their platform.

**Comment by arnabk** [comment]
But they do allow cordova, which is the same concept that Firefox is using.

**Comment by arnabk** [comment]
JY just sent me this link. https://support.mozilla.org/en-US/kb/is-firefox-available-iphone-or-ipad

**Comment by borisyankov** [comment]
Yeah, but Firefox apps use their custom framework XUL, that will unlikely be available.
The same with Chrome - I use it on iOS mainly for the login/bookmarks sync, but since it wraps webkit, it has only the features Safari has, and nothing more.
